### PR TITLE
Fix UI :   Fix  password validation while creating account

### LINKED ISSUE #7779: UI: Add password validation to Register New Account page
**Affected module**
UI

**Describe the bug**

- [ ] There is no password validation in the UI on the Register New Account page. It allows me to create an account with a password less than 8 characters.

- [ ] The form should not be cleared until a success response from the API. 

Screen Shot 2022-09-28 at 9 51 59 AM

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue :- Closes #7779
- Fix  password validation while creating account

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
 @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/signup/basic-signup.component.tsx [hunk 1]
```diff
@@ -111,17 +112,14 @@ const BasicSignUp = () => {
...
-
-                                return Promise.resolve();
-                              },
+                              required: true,
+                              message: 'Password is required',
+                            },
+                            {
+                              pattern: passwordRegex,
+                              message:
+                                'Password must be of minimum 8 and maximum 16 characters, with one special , one upper, one lower case character',
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/pages/signup/basic-signup.component.tsx:122]:
```suggestion
                                'Password must be a minimum of 8 and a maximum of 16 characters long and contain at least one uppercase and one lowercase letter (A, z), and one special character (such as !, %, @, or #)',
```

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/pages/signup/basic-signup.component.tsx]: @vivekratnavel  Updated!

**Comment by chirag-madlani** [comment]
> image
@Ashish8689 Let's show one error at a time.
In the image above we are showing two
1. Confirm password is required - We can avoid this as second rule will make sure that it's present
2. Password doesn't match

**Comment by Ashish8689** [comment]
@chirag-madlani  Updated !

**Comment by Ashish8689** [comment]
@chirag-madlani  Updated !

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**